vrx_gazebo/src/follow_plugin.cc

```diff
@@ -64,7 +70,66 @@ void FollowPlugin::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf)
       waypointElem = waypointElem->GetNextElement("waypoint");
     }
   }
+  // If no waypoints present, check for the <circle> element and parse.
+  else if (_sdf->HasElement("circle"))
+  {
+    gzmsg << "Circle element activated" << std::endl;
+    auto circleElem = _sdf->GetElement("circle");
+    double radius = 5;
+    // Parse the optional <radius> field.  If absent, use the default (5).
+    if (circleElem->HasElement("radius"))
+    {
+      auto radElem = circleElem->GetElement("radius");
+      radius = radElem->Get<double>();
+    }
+    // Get the current model position in global coordinates.  Create
+    // local vectors that represent a path along a rough circle.
+    ignition::math::Vector2d position(this->modelPose.Pos().X(),
+                                      this->modelPose.Pos().Y());
+    double angle = 0;
+    ignition::math::Vector2d vec(radius / 2, 0);
```

Comment:
@j-herman , I know we merge this but does it make sense `radius / 2` here and in line 97?  I think it should be just radius, right?

Reply:
Oh, no!  I started out with circumference measurements in the initial version and never updated the equation.  I'm inclined to fix it by changing the variable name and documentation back, so nothing will change with the existing worlds - sound ok?

Reply:
> Oh, no! I started out with circumference measurements in the initial version and never updated the equation. I'm inclined to fix it by changing the variable name and documentation back, so nothing will change with the existing worlds - sound ok?

So converting it to `diameter`?

Reply:
Diameter.  Right.  Not circumference.

Reply:
Sounds good to me!